adapters/33across/33across_test.go

```diff
@@ -4,8 +4,17 @@ import (
 	"testing"
 
 	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
 func TestJsonSamples(t *testing.T) {
-	adapterstest.RunJSONBidderTest(t, "33across", New33AcrossBidder("http://ssc.33across.com"))
+	bidder, buildErr := Builder(openrtb_ext.Bidder33Across, config.Adapter{
+		Endpoint: "http://ssc.33across.com"})
+
+	if buildErr != nil {
+		t.Fatalf("Builder returned expected error %v", buildErr)
```

Comment:
I believe this should be `"Builder returned unexpected error %v"`, unless you are extremely pessimistic about the quality of adapters in this project. This typo is copy and pasted across all the adapter tests it would appear.

Reply:
LOL. Nice catch. I'll fix the typo.